src/mmw/js/src/modeling/views.js

```diff
@@ -209,10 +210,15 @@ var ProjectMenuView = Marionette.ItemView.extend({
     saveProjectOrLoginUser: function() {
         if (App.user.get('guest')) {
             var self = this;
-            App.getUserOrShowLogin(function() {
-                self.model.setUserIdOnProjectAndScenarios();
-                self.model.saveInitial();
-            });
+
+            new modalViews.CreateAccountPersuasionModalView({
```

Comment:
Not sure how common this use case is, but previously because we were using `getUserOrShowLogin`, if the user had signed in from another tab, clicking "☁️ Save" would not show any modal, it would simply log the user in and save the project.

Now, the modal will always be shown, even if the user is logged in, and they'll have to click "Login" which will do the same thing.

I don't think this should be too bothersome to the small percentage of users that encounter this.